stock_location_limit_product/models/stock_location_limit.py

```diff
@@ -1,33 +1,35 @@
 # Copyright (C) 2019 Open Source Integrators
 # Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
-
-from odoo import fields, models
+from odoo import api, fields, models, _
 from odoo.addons import decimal_precision as dp
+from odoo.exceptions import ValidationError
 
 
 class StockLocationLimit(models.Model):
     _name = 'stock.location.limit'
     _description = 'Stock Location Limit Product'
     _rec_name = 'product_id'
 
+    @api.onchange('product_id')
+    def onchange_uom_id(self):
```

Comment:
I removed the onchange of product_id because as per @rousseldenis comment's on [PR:757](https://github.com/OCA/stock-logistics-warehouse/pull/787) made uom_id as the related field.